omod-1.10/src/test/java/org/openmrs/module/attachments/rest/AttachmentController1_10Test.java

```diff
@@ -234,4 +235,28 @@ public void postAttachment_shouldUploadFile() throws Exception {
 			Assert.assertNull(obs.getEncounter());
 		}
 	}
+	
+	@Test(expected = IllegalRequestException.class)
+	public void postAttachment_shouldNotUploadFileAboveSizeLimit() throws Exception {
+		
+		String fileCaption = "Test file caption";
+		
+		// Setup
+		String fileName = "testFile1.dat";
+		Patient patient = Context.getPatientService().getPatient(2);
+		Visit visit = Context.getVisitService().getVisit(1);
+		
+		byte[] maxData = new byte[(int) ((attachmentsContext.getMaxUploadFileSize()*1024*1024)+1)];
+		
+		MockMultipartHttpServletRequest request = newUploadRequest(getURI());
+		MockMultipartFile file = new MockMultipartFile("file", fileName, "application/octet-stream", maxData);
+		
+		request.addFile(file);
+		request.addParameter("patient", patient.getUuid());
+		request.addParameter("visit", visit.getUuid());
+		request.addParameter("fileCaption", fileCaption);
+		
+		// Reply
```

Comment:
`// Replay`

By the way, if you came across this typo elsewhere, would you mind correcting it there too?